gui/wxpython/nviz/wxnviz.py

```diff
@@ -411,7 +413,7 @@ def GetViewpointPosition(self):
 
         return (x.value, y.value, h.value)
 
-    def LookHere(self, x, y, scale=1):
+    def LookHere(self, x, y, scale: float = 1) -> None:
```

Comment:
Why not annotate x and y and why is scale float but as int as default?

Reply:
I wasn't able to trace back (yet) what it was, if it can be float, or only ints. If i remember well, this and another set of functions that originate from events of wxPython, I couldn't determine if we expected floats (like parts where some screen coordinates seem to be transformed to another reference system, but in the same function it is scaled and truncated to ints, so the link is kinda broken. I hope that with more info, later on, these uncertainties won't be there

Reply:
For the scale, it is absolutely for sure float, as it comes from the GetContentScaleFactor(), that I did a full research on it  a couple days ago and understanding how they wanted us to use these functions, to have something like this:
![image](https://github.com/user-attachments/assets/cb072c0d-cea1-44bb-801e-6015a88f6a25)

(That rabbit hole was kinda deep, and quite long to get the answer with respect to the size of the question)